system-tests/lib/cre/environment/fund_cl_nodes_op.go

```diff
@@ -57,6 +59,21 @@ var FundCLNodesOp = operations.NewOperation[FundCLNodesOpInput, FundCLNodesOpOut
 				}
 				for _, node := range metaDon.DON.Nodes {
 					errGroup.Go(func() error {
+						if bcOut.SolChain != nil {
+							funder := bcOut.SolChain.PrivateKey
+							recipient := solana.MustPublicKeyFromBase58(node.AccountAddr[bcOut.SolChain.ChainID])
+
+							err := libfunding.SendFundsSol(ctx, zerolog.Logger{}, bcOut.SolClient, libfunding.FundsToSendSol{
+								Recipent:   recipient,
+								PrivateKey: funder,
+								Amount:     50_000_000,
+							})
+							if err != nil {
+								return fmt.Errorf("failed to fund sol node: %w", err)
+							}
+							return nil
+						}
+
```

Comment:
maybe add also info level messages at the beginning and at the end, so that we know that funding for each node was done?